views/werdz.erb

```diff
@@ -0,0 +1,29 @@
+<h1> Score all the werdz </h1>
+
+<h3>Enter your scrabble words below to get the scores for all of your words and each of their letters.</h3>
+<p>Enter your words separated by a comma with no spaces
+For example, if your words are 'hi', 'hello', and 'bye'
+enter in hi,hello,bye in the textbox below and then click 'Submit'.</p>
+
+<form action="" method="post" accept-charset="utf-8">
+  <fieldset>
+    <legend>Words to Score</legend>
+    <p><textarea rows="4" cols="50" placeholder="type,your,words,separated,by,commas,like,this (no spaces)" name="words[]"></textarea></p>
+    <input type="submit" value="Submit">
+  </fieldset>
+</form>
+
+
+<% @words.each do |word| %>
+  <% points = Scrabble::Scrabble.score(word) %>
```

Comment:
Generally it's good to avoid calling directly to your classes in your views - It might make more sense to call this in your controller method and pass the result into the view.